packages/leaflet-map/src/base-map.tsx

```diff
@@ -241,7 +242,9 @@ const BaseMap = ({
           ) : null}
 
           {!!currentMarkers && currentMarkers.length > 0 && currentMarkers.map((data) => {
-            if (data.isClustered) {
+            // temp bug fix
+            // if (data.isClustered) {
+            if (0 && data.isClustered) {
```

Comment:
Kan deze tijdelijke bugfix — met de PRs van Niels omtrent de map — er nu uit? Of werkt het dan nog niet?

Reply:
Yes, dat kan eruit inderdaad!